integrations/ScriptLoader.js

```diff
@@ -4,15 +4,25 @@ import logger from "../utils/logUtil";
 const defaultAsyncState = true;
 
 export const LOAD_ORIGIN = "RS_JS_SDK";
-
+/**
+ * Script loader
+ * @param {String} id                       Id of the script
+ * @param {String} src                      URL of the script
+ * @param {Object} options                  Object containing different configuaration
+ * @param {Boolean} options.async           Determines script will be loaded asynchronously or not
+ * @param {Boolean} options.isNonNativeSDK  Determines whether the script that will be loaded is one of RS's own
+ * @param {Boolean} options.polyfillLoad    Determines ScriptLoader is called for loading the polyfill
```

Comment:
Rename it to `skipDatasetAttributes`. Add comments for establishing the use case.

Reply:
Adressed